### ISSUE TITLE
please update its not owekring for us

### ISSUE THREAD

**Issue by fisforfaheem** [demj1206170/EnableSoli#7]
please

**Comment by demj1206170** [comment]
it‘s still working on TP1A.22.1005.002.B2 ,have you. correct install the Magisk app and get device rooted?

**Comment by fisforfaheem** [comment]
yes, tried all things, i have android 13 pixel 4xl, it enables soli but soli fucntiosn not work even if enabled
